packages/http-client/src/index.d.ts

```diff
@@ -4,17 +4,18 @@ import type {
   UserProp,
 } from "@webstudio-is/react-sdk";
 
-export type IncludeTypes = "tree" | "props" | "breakpoints" | "pages";
-export type Includes<T> = { [include in IncludeTypes]: T };
 export type Page = {
   id: string;
   root: [string, string];
 };
 export type Project = {
-  tree: BaseInstance & { errors: string };
-  props: Array<UserProp> & { errors: string };
-  breakpoints: Array<Breakpoint> & {
-    errors: string;
+  [include in string]: {
+    page: Page;
+    tree: BaseInstance & { errors: string };
+    props: Array<UserProp> & { errors: string };
+    breakpoints: Array<Breakpoint> & {
+      errors: string;
+    };
```

Comment:
From the consumer of REST api point of view, is it useful to get a response where you get all the data, but a `{ errors: string }` in one of the fields? Maybe the response type should be either all data or an error?